cprover_bindings/src/irep/goto_binary_serde.rs

```diff
@@ -0,0 +1,1339 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+//! GOTO binary serializer.
+//! #![allow(dead_code, unused)]
+use memuse::DynamicUsage;
+
+use crate::irep::{Irep, IrepId, Symbol, SymbolTable};
+use crate::{InternString, InternedString};
+use std::collections::HashMap;
+use std::fs::File;
+use std::hash::Hash;
+use std::io::{self, BufReader};
+use std::io::{BufWriter, Bytes, Error, ErrorKind, Read, Write};
+use std::path::PathBuf;
+
+/// Writes a symbol table to a file in goto binary format in version 5.
```

Comment:
Any link we can add to this version of the goto binary format?

Reply:
The best we can do for now is a reference to a source file in CBMC